### DISCUSSION TITLE
Plane View

### DISCUSSION THREAD

**Discussion by El-Helbawy-M** [sceneview/sceneview#380] [Q&A]
How can I change the "textures/plane_renderer.png" that is used with the PlaneRanderer ? 

And, can I raise the accuracy of the plane detection ?

**Comment by ThomasGorisse** [comment]
If you're looking to render a flat plane in v4.0.0-rc.1, use the geometry nodes:

```kotlin
SceneView {
    // Flat plane using a cube with very small height
    CubeNode(
        materialInstance = material,
        size = Size(2f, 0.01f, 2f) // wide and flat
    )
    
    // Or use ImageNode for a textured plane
    ImageNode(
        materialLoader = materialLoader,
        imageFileLocation = "textures/floor.png",
        width = 2f, height = 2f
    )
}
```

For AR plane visualization (detected surfaces), `ARSceneView` handles plane rendering automatically.
